requirements/adg/base.txt

```diff
@@ -1 +1,3 @@
 mandrill==1.0.57
+phonenumbers==8.10.0
+django-phonenumber-field==4.0.0
```

Comment:
Let's please make sure that these new packages are not downgrading or upgrading any existing package.

Reply:
@javeriaejaz-arbisoft Have you check this. To verify it, you have to pip freeze, before and after installing these packages and see nothing else has changed.